### PR TITLE
`network` - add screen to change network outside of wallet context

### LINKED ISSUE #566: `network` - select outside of wallet context
#### What would you like to be added:
Currently, when user is out of wallet context (e.g, Onboarding, restore flow). User doesn't have a way to select networks. He needs to create a wallet first and is currently a funnel.

- [ ] Add on Create Wallet Guidelines (First Screen only)
- [ ] Restore page

#### Why is this needed:

### PR DESCRIPTION
**Pull request by kyleleow**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
- To provide an option to user to switch network during create wallet and restore wallet flow

#### Which issue(s) does this PR fixes?:

Fixes #566

#### Additional comments?:
Replace bottom modal used in mock UI with a screen due to technical difficulty

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

**Comment by thedoublejay** [comment]
@kyleleow we need to have a different alert message when you're out of wallet context since you're already on the onboarding screen.

**Comment by kyleleow** [comment]
@thedoublejay For the new message, can I go with: "You are about to switch to {{network}}. Do you want to proceed?"

**Comment by thedoublejay** [comment]
Ok @kyleleow, you can ask @izzycsy for the german translations also

**Comment by kyleleow** [comment]
Added german translation from Bella

**Review by thedoublejay — CHANGES_REQUESTED** [review]
1. Error on message when switching both on outside of wallet context and from settings page. Network selecting is wrong. Probably a good idea to add an e2e on this.

**Comment by thedoublejay** [comment]
Also, one more minor change. Can you replace the icon only on Guidelines page and Restore page? Just to signify that the network can be selected. You can check with @jtlntno-cake on the icon class name.
Screenshot 2021-09-02 at 4 57 34 PM

File: app/components/HeaderTitle.tsx [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+      <ThemedText
+        dark={tailwind('text-white text-opacity-90')}
+        light={tailwind('text-black')}
+        style={tailwind('font-semibold leading-5')}
+        testID={testID}
+      >
+        {text}
+      </ThemedText>
+
+      {subHeadingType === 'None' && <></>}
```
Comment (ivan-zynesis) [inline app/components/HeaderTitle.tsx:27]: This is redundant right? undefined should cover it.

File: app/screens/WalletNavigator/screens/CreateWallet/OnboardingNetworkSelectScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  return (
+    <View testID='onboarding_network_selection_screen'>
+      <SectionTitle
+        testID='onboarding_network_selection_screen_title'
+        text={translate('screens/OnboardingNetworkSelectScreen', 'SELECT NETWORK')}
+      />
+
+      {
+        networks.map((network, index) => (
+          <RowNetworkItem
```
Comment (ivan-zynesis) [inline app/screens/WalletNavigator/screens/CreateWallet/OnboardingNetworkSelectScreen.tsx:20]: I actually prefer just make another simple component here.

1. This is not part of Settings screen anyway
2. Wish this page page auto dismiss after selected a network (my personal preference on the UX)
3. The original row component will has fixed alert message and tidier

Comment (kyleleow) [reply app/screens/WalletNavigator/screens/CreateWallet/OnboardingNetworkSelectScreen.tsx:20]: After consulting Jonathan, he mentioned that the network selection UI will have changes again in the future and both screen (onboarding and setting) should always be the same.

Can I still go with the current solution which is to share the same `RowNetworkItem` component, but I'll move it to `app/components` ?

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**